tests/guestlog/guestlog.go

```diff
@@ -244,3 +295,16 @@ func generateHugeLogData(vmi *v1.VirtualMachineInstance, mb int) {
 		&expect.BExp{R: "logline " + endn},
 	}, 240)).To(Succeed())
 }
+
+func getConsoleLogs(virtClient kubecli.KubevirtClient, virtlauncherPod *k8sv1.Pod) (string, error) {
+	logsRaw, err := virtClient.CoreV1().
+		Pods(virtlauncherPod.Namespace).
+		GetLogs(virtlauncherPod.Name, &k8sv1.PodLogOptions{
+			Container: "guest-console-log",
+		}).
+		DoRaw(context.Background())
+	if err != nil {
```

Comment:
You can just do `return string(logsRaw), err`

Reply:
done